### DISCUSSION TITLE
Writers Guild Governance Proposal: Improve Elections Processes and Clarify Candidate Responsibilities and Expectations

### DISCUSSION THREAD

**Discussion by lvnacy** [BanklessDAO/writers-guild-governance-framework#1] [Governance]
Author: nonsensetwice#3475
Date: January 23, 2022

## TABLE OF CONTENTS
1. [Definitions](#definitions)
2. [Background](#background)
3. [Reasoning](#reasoning)
4. [Solution & Implementation](#solution-implementation)

## DEFINITIONS

As of this writing, the nomenclature for the Writers Guild (hereafter, the Guild) tiered membership system is still being considered. We want to have more fun with it, but have not settled on what that fun looks like. Rather than create confusion around tier names, the tiers will be simplified as follows:

- Tier 1 represents the lowest membership, or entry level tier, what is currently referred to as “Pupil” in the Governance Framework;
- Tier 2 represents the middle tier, which is currently referred to as “Scribbler; and
- Tier 3 represents the full membership tier, which is currently referred to as “Scribe.”

## BACKGROUND

A tie took place during the most recent election for Guild Treasurer, with concerns being raised over the ethics of candidates voting for themselves to win the election. In the case of this tie, one candidate voted for themselves, whereas the other did not, implying that had the former not voted for themselves, or had the latter voted for themselves, there would have been a clear winner. This proposal seeks to clarify the position the Guild takes with regard to a candidate’s self-interest, and provide an elections code of conduct.

## REASONING

“Power attracts the corruptible. Absolute power attracts the absolutely corruptible. This is the danger of entrenched bureaucracy to its subject population.”
— Frank Herbert, Dune Messiah

When the Guild was founded and efforts were undertaken to produce the governing document for the Guild, a great amount of emphasis was placed on the importance of limiting any one member’s ability to centralize power by any bureaucratic method or means. This resulted in the current standards for candidate eligibility, namely a Tier 3 member in good standing and a candidate selection process that discourages self-nomination. In alignment with this ideology, the Guild should adopt a stance that further discourages self-interest by disqualifying candidates who vote for themselves during a general election, an off-cycle election, and any run-off election that takes place.

It is important to note that the intended goal of an election is to allow the Tier 3 members of the Guild to collectively decide who they think will perform best in a given role, with zero regard to how deeply a candidate seeks to serve in that role. The candidate’s role in the selection process is accept a given nomination, and provide the content necessary for Guild members to make the most qualified and informed decision. It is at this point that a candidate’s role in the election ends, with the exception of answering questions raised by voting members.

A candidate running on good faith with the intention to serve the best interests of the Guild will do nothing to sway the election in their favor, including, but not limited to:

- vote for themselves
- publicly and/or privately campaign for themselves
- convince fellow Tier 3 members to cast a vote in their favor
- work to shift fellow Tier 3 members’ sentiment against opponents
- slander, belittle, or insult opponents
- _conduct themselves without honesty or respect for the process_ (edit provided by @trewkat)

Furthermore, as elections are aimed at providing the Guild with the best person to serve in each role, votes should be cast with regard to a candidate’s merits and not their popularity. As such, it is every Tier 3 member’s responsibility to withhold judgement on a candidate’s character where such character does not factor into a role and its responsibilities. When this practice is taken seriously by all members and candidates, all candidates can celebrate any election’s outcome as a win for the Guild, whether they win the election or not.

## SOLUTION & IMPLEMENTATION

Upon ratification of this proposal, the Writers Guild Governance Framework will be updated to clarify language with regard to self-interest in **Article VII Elections**, and a Candidate’s Code of Conduct will be drafted and submitted to the Guild for review, comments, and approval.

**Comment by hirokennelly** [comment]
Great work nonsense. The body of the doc is fantastic. Per your comments re: title, I'd suggest something like "Candidate's Code of Conduct" or similar language

**Comment by trewkat** [comment]
**_"the Guild should adopt a stance that further discourages self-interest by disqualifying candidates who vote for themselves during a general election, an off-cycle election, and any run-off election that takes place."_**

Can I please clarify? Is this section intended to say that the candidate themselves would be disqualified if they vote for themselves, or just that their vote would be disqualified i.e. not counted? It seems overly harsh to disqualify someone from candidature based on a vote for themselves.

Regardless of intended meaning there, I disagree with the idea that candidates should not vote in the election. It is pretty standard practice for candidates to vote for themselves in an election. Politicians do it, and nobody would expect them to vote for somebody else, although if they were to do so, that is their right.
By denying candidates the right to vote in an election that they are running in, we would be denying them the right to participate in the governance process.

_[I say this as someone who has routinely chosen not to vote for myself or any other candidate in various role elections / consensus votes for which I have been a candidate, on the basis that it felt a bit unethical in such small polls, especially if the vote was anonymous. I am _also_ someone who has routinely voted for my own POAP design when those polls were happening in the DAO.]_

I agree with the sentiment of the other listed actions we'd like to prevent through our governance documentation, but I don't agree that removing the ability to vote is in the spirit of open governance. Instead, I think the documentation should explicitly say that candidates are encouraged to vote and we should make sure all candidates are aware of that prior to the poll.

I also oppose the term "_exhibit unsportsmanlike conduct_" and I suggest '_exhibit unsporting behavior_', or even better, '_conduct themselves without honesty or respect for the process_' as more inclusive and plainer English alternatives to that gender-specific wording.

Lastly, I believe the sentence "_The candidate’s role in the selection process is accept a given nomination_" needs the word '_to_' added before the word '_accept_'.

**Reply by Coolcatcatio** [reply]
I agree that candidates should be able to participate in voting. As Trewkat said, it's standard as far as I am aware, and candidates' votes often cancel each other out as long as they understand they are all eligible and expected to vote (like everyone is expected to vote). An issue would definitely come up if the votes were weighted differently depending on BANK holdings, but this isn't the case here.

**Reply by hirokennelly** [reply]
clearly whether a person can vote for themselves is where the contention is going to be, and very much understandably so, as it's at the intersection of agency, autonomy, liberty, and self interest. JUICY AF.

I personally find candidates voting for themselves just (shhuddderrrs), and I don't think just b/c we do it in fiat space means we should port terrible ideas over to the cryptoverse. 

It's a great point about open governance, but at the same time, it's a practice I would like to see abolished across the board, along w/ self nomination. sadly, based upon other guilds, not likely anytime soon :( 

Perhaps for now, the check and balance within the ban on self nomination is enough for most folks, but I'd prefer to see it banned.

**Reply by lvnacy** [reply]
I agree with @hirokennelly on this point regarding candidates voting for themselves. We can have open governance and let candidates watch the process unfold without their intervention. These two things are not mutually exclusive.

Thank you for the review of terminology, @trewkat. I will update the document with your suggestions pertaining to better gender-neutral phrasing.

**Reply by AustinFoss** [reply]
I could see it go either way whichever way the guild decides for self voting, it just needs to be made clear which one it is. I do share Hiro's sentiment about self-voting, it just gives me a bad feeling inside. Ultimately it has no impact on the election whichever way it goes because it cancels out, just clarity is missing as to which it is. I don't feel like not being able to vote as a candidate means I'm not able to participate in governance because running as a candidate feels like participation to me.

Self-nomination should certainly be a no-no.

Regarding the rest of the suggestion about the code of conduct:

1. vote for themselves
   - Agreed but ultimately am impartial
1. publicly and/or privately campaign for themselves
   - Agreed
1. convince fellow Tier 3 members to cast a vote in their favor
   - Agreed
1. work to shift fellow Tier 3 members’ sentiment against opponents
   - Agreed
1. slander, belittle, or insult opponents
   - Agreed
1. exhibit unsportsmanlike conduct
   - Agreed

5 & 6 seem pretty self explanatory. The only thing that's maybe a little too ambiguous is what is "unsportsmanlike conduct". I like to put everything in terms of math and numbers, a formula, but this one seems like it has to remain somewhat ambiguous. Maybe just a process for determining this (a second vote by scribes to determine if this occurred)?

1 is the one that's caused this mess.

2, 3, & 4 I'd like to give an explanation why I agree with no campaigning behavior. It seems to me that given that "This proposal seeks to clarify the position the Guild takes with regard to a candidate’s self-interest" and "as elections are aimed at providing the Guild with the best person to serve in each role, votes should be cast with regard to a candidate’s merits and not their popularity." Speaking in-terms of my own self interest, my agreement is honestly because of my own personal strengths; I'm not good at "campaigning".

Personally, I feel like, and this is how I vote IRL a lot of the time, it's best to disregard everything a candidate says once they've announced they're running. Things shift from 'doing' to 'what will be done', leading to a lot of 'talking big'. I feel like a person's campaign should just be a record of what they've achieved as a predictor for what they'll do in office. Perhaps at a larger scale, like nations, when average people need that refresher on who's running once every 4 years, campaigns make sense to get an idea what the platform is (but then the debates are just horrible anyway). But at this scale a campaign/resume/application for the role should just be the body of work done up to that point, the experiences and impressions left on our coworkers should speak for itself without the adulteration of candidates trying to get people to vote for them.

Late last week I could feel ideas tugging at my brain when the race was close, "maybe I should dm X and remind them to vote" (not necessarily for me, just a reminder to vote, but even that carries the implication who I want them to vote for). Maybe if we had anonymous ballots that'd change, maybe (i'd have to consider that a bit more), but given we know who votes for who, asking people to vote puts a different kind of pressure on them.

But that's playing to my strengths, and it's not to say that's how everyone should vote. And maybe that'd take away from another scribe who wants to vote based on the information in a campaign, or does a disservice to a candidate who doesn't make those connections with other contributors during the season and is just head down working all the time and so would benefit from the chance to campaign (but I believe in that case the rest of the scribes would probably just organically promote such a dedicated contributor anyway). Plus there's no way to get around private DM's and campaigning that way, just left with trust to not do that.

A bit long winded here, but that's my thoughts and explanation for them.

**Comment by thinkDecade** [comment]
Interesting. 👍🏾. here to observe.  not an active member of the guild but I love the ideas in the proposal and responses.

**Comment by 0xblur** [comment]
Interesting observations, I’m not familiar with the controversy, but I would like to bring what I think to the table.

If roles involve capability more than any other type of qualification, like trust, involvement, or any social status you can obtain DAOing, then an election process wouldn’t be necessary. For that, a qualification assessment would suit better. 

I think we are trying to dismiss an inevitable part of any political process: public displays. Even if you limit the narcissistic displays candidates can make during the election process, you can’t elude the ones made in more concealed scenarios. There will be people that have amassed reputation over time, and they will have an advantage over young, less socially-involved members. Even if the new face is more qualified in education, and experience to suit the role best, [s]he wouldn’t receive the deserved attention. Limiting the displays of the less socially recognized candidate will benefit the incumbent, especially if [s]he hasn’t marketed himself in previous set-ups.

Following this logic, I think ‘self-interest’ is getting a bad connotation here. Convincing others is a way to ask for an opportunity, no more, no less. Otherwise, the incumbents will stay in power, which is not inherently bad; nonetheless, a chance to say ‘I can do things better’ appears necessary.

**Comment by trewkat** [comment]
"_**Upon ratification of this proposal, the Writers Guild Governance Framework will be updated to clarify language with regard to self-interest in Article VII Elections**, and a Candidate’s Code of Conduct will be drafted and submitted to the Guild for review, comments, and approval._"

How will this proposal be ratified? Are you looking at holding a vote?
I suggest that will be necessary given the views expressed in relation to voting by candidates.

It's not clear in this post whether the language used in the Reasoning section is the exact language proposed for the governance document, but as far as I can tell this is just for discussion and then Scribes would be asked to vote on the exact update.
Is that correct?

**Reply by lvnacy** [reply]
> How will this proposal be ratified? Are you looking at holding a vote? I suggest that will be necessary given the views expressed in relation to voting by candidates.

Yes, this will eventually go to a vote, once we are agreed on the text and the proposed changes, which we should all shape together.

> It's not clear in this post whether the language used in the Reasoning section is the exact language proposed for the governance document, but as far as I can tell this is just for discussion and then Scribes would be asked to vote on the exact update. Is that correct?

Correct, this is just for discussion. The idea is that proposal is just a draft, and together we hone it to include the text that we would like included in the framework.
